src/main/kotlin/no/skatteetaten/aurora/boober/service/DeployService.kt

```diff
@@ -103,10 +105,17 @@ class DeployService(
         }
 
         val openShiftApplicationObjects: List<JsonNode> = openShiftObjectGenerator.generateApplicationObjects(deploymentSpec, deployId)
-        val openShiftApplicationResponses: List<OpenShiftResponse> = openShiftApplicationObjects.map {
+        val openShiftApplicationResponses: List<OpenShiftResponse> = openShiftApplicationObjects.flatMap {
```

Comment:
All changes in this file should probably be put in createOpenShiftCommand method - which is now in danger of becoming very complex itself. Consider creating a new class for it.